packages/components/src/components/select-native/select-native.jsx

```diff
@@ -15,6 +15,16 @@ const getDisplayText = (list_items, value) => {
     return '';
 };
 
+const getOptions = (list_items, hide_disabled, use_text) => {
```

Comment:
Since this returns a component you can name and use it as such, e.g.

```suggestion
const SelectNativeOptions = ({ list_items, hide_disabled, use_text }) => {
    // Insert component definition here.
}
```

Then later you could use it as per below:

```JS
<SelectNativeOptions 
    list_items={list_items}
    hide_disabled={hide_disabled}
    use_text={use_text}
/>
```